docs/src/main/asciidoc/rest-client-reactive.adoc

```diff
@@ -1364,6 +1364,8 @@ quarkus.rest-client.logging.body-limit=50
 quarkus.log.category."org.jboss.resteasy.reactive.client.logging".level=DEBUG
 ----
 
+REST Client Reactive uses a default `ClientLogger` implementation. You can change it by providing a custom `ClientLogger` instance via CDI or when programmatically creating your client.
```

Comment:
I will add this sentence as a TIP. Also, fix the reported issue by Linting with Vale.

Reply:
>I will add this sentence as a TIP.
Sorry to ask, but what do you mean by that?

I will fix the linting issue.